client/src/components/Search.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react";
+import ListCourse from "./ListCourse";
+
+class Search extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { initial: this.props.items, current: this.props.items };
+    this.filterItems = this.filterItems.bind(this);
+  }
+
+  filterItems(input) {
+    var updatedlist = this.state.initial;
+    updatedlist = updatedlist.filter(item => {
+      return (
+        item.code.toLowerCase().search(input.target.value.toLowerCase()) !== -1
```

Comment:
Maybe just change this to work with course name as well.

Reply:
ok